.github/workflows/tests.yml

```diff
@@ -19,13 +19,22 @@ jobs:
       - name: Start Docker container with localstack
         if: matrix.session == 'localstack'
         run:  docker run -d -p 4566:9324 --rm softwaremill/elasticmq-native
+
       - name: Checkout Repo
         uses: actions/checkout@v3
+
       - name: Set up Python
         uses: actions/setup-python@v2
```

Comment:
Very minor nit: the "publish" workflow uses `setup-python@v4`, for the sake of consistency we should probably use the same here.